modules/ml/src/main/java/org/apache/ignite/ml/KMeansModelFormat.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.ignite.ml;
+
+import java.io.Serializable;
+import java.util.Arrays;
+import org.apache.ignite.ml.math.DistanceMeasure;
+import org.apache.ignite.ml.math.Vector;
+
+/**
+ * K-means model representation.
+ *
+ * @see Exportable
+ * @see Exporter
```

Comment:
it's hard to tell how "@see Export/able" references are relevant here. In order to help reader see the connection consider either extending first sentence to explain that representation is used for export or replacing these see-references with something like "@see org.apache.ignite.ml.clustering.KMeansModel#saveModel"

Reply:
Currently this is intro for model import/export mechanism and I think that this javadoc good enough for now.